app/src/lib/git/commit.ts

```diff
@@ -22,23 +29,53 @@ export async function createCommit(
     })
     return true
   } catch (e) {
-    // Commit failures could come from a pre-commit hook rejection. So display
-    // a bit more context than we otherwise would.
-    if (e instanceof GitError) {
-      const output = e.result.stderr.trim()
+    handleCommitError(e)
+    return false
+  }
+}
+
+/**
+ * Creates a commit to finish an in-progress merge
+ * assumes that all conflicts have already been resolved
+ *
+ * @param repository repository to execute merge in
+ * @param files files to commit
+ */
+export async function createMergeCommit(
```

Comment:
i'm open to better names for this if you have one

Reply:
Nah, this works for me 👍